websub-ballerina/src/websub/subscriber_client.bal

```diff
@@ -118,13 +118,11 @@ isolated function buildSubscriptionChangeRequest(@untainted string mode,
 #            if an error occurred
 function processHubResponse(@untainted string hub, @untainted string mode,
                             SubscriptionChangeRequest subscriptionChangeRequest,
-                            http:Response|error response, http:Client httpClient,
+                            http:Response|http:Payload|error response, http:Client httpClient,
                             int remainingRedirects) returns @tainted SubscriptionChangeResponse|error {
 
     string topic = subscriptionChangeRequest.topic;
-    if (response is error) {
-        return WebSubError("Error occurred for request: Mode[" + mode+ "] at Hub[" + hub + "] - " + response.message());
-    } else {
+    if (response is http:Response) {
```

Comment:
Here we can keep the #125 to #127 and cast the response to `http:Response` in the else part. WDYT?